src/mavlink-router/ulog.cpp

```diff
@@ -167,17 +167,24 @@ int ULog::write_msg(const struct buffer *buffer)
         if (trimmed_zeros)
             memset(((uint8_t *)&cmd) + payload_len, 0, trimmed_zeros);
 
-        if (!_logging_start_timeout || cmd.command != MAV_CMD_LOGGING_START)
+        if ((!_logging_start_timeout || cmd.command != MAV_CMD_LOGGING_START) &&
+                (!_logging_stop_timeout || cmd.command != MAV_CMD_LOGGING_STOP))
             return buffer->len;
 
         if (cmd.result == MAV_RESULT_ACCEPTED) {
-            _remove_start_timeout();
-            if (!_start_alive_timeout()) {
+            if (_logging_start_timeout && cmd.command == MAV_CMD_LOGGING_START) {
+                _remove_start_timeout();
+
+                if (!_start_alive_timeout()) {
                 log_warning("Could not start liveness timeout - mavlink router log won't be able "
```

Comment:
Can you run the style check? -> https://github.com/Auterion/mavlink-router#contributing

Reply:
@bkueng Done